src/deepsparse/v2/text_generation/nl_engine_operator.py

```diff
@@ -36,6 +36,7 @@ class NlEngineInput(BaseModel):
     engine_inputs: List = Field(description="engine inputs")
     kv_cache: Any = Field(description="kv_cache object")
     tokens: List = Field(description="tokens")
+    in_generation: bool = Field(description="in_generation", default=None)
```

Comment:
This is roughly the final version of the code, right? Let's make sure that the descriptions are informative.

Reply:
No, not until we've until tested everything.